app/models/contact.rb

```diff
@@ -1,2 +1,14 @@
 class Contact < ApplicationRecord
+  dragonfly_accessor :image
+
+  #image validations
+  #
+  validates_presence_of :image
+  validates_size_of :image, maximum: 200.kilobytes,
+                    message: "should not be more than 400KB", if: :image_changed?
```

Comment:
Here you set a maximum size of the picture to 200 kilobytes, but wrote 400. I think here should be at least 400, maybe more